front/pages/api/poke/workspaces/[wId]/conversations/[cId]/index.ts

```diff
@@ -43,18 +44,14 @@ async function handler(
 
   switch (req.method) {
     case "GET":
-      const conversation = await getConversation(auth, cId, true);
+      const conversationRes = await getConversation(auth, cId, true);
```

Comment:
Here, we do this check for all the routes below ( > 10 times)
We could do it via wrapper. A bit less readable, but more compact.
Open to refactoring it to a wrapper in a follow-up pr

Reply:
I think that's fine to be explicit here with the apiErrorForConversation you introduced limiting code :+1: